packages/i18nify-js/src/modules/shared/types.d.ts

```diff
@@ -0,0 +1,3 @@
+import { COUNTRY_CODES } from './constants';
+
+export type CountryCodeType = typeof COUNTRY_CODES[number];
```

Comment:
create only types file, we don't need constants file. 

export type CountryCodeType = 'US' | 'IN'... 

Something like this or a better way.